src/DeviceRow.vala

```diff
@@ -176,6 +178,33 @@ public class Bluetooth.DeviceRow : Gtk.ListBoxRow {
         });
     }
 
+    private void config_settings_button () {
```

Comment:
Is there a reason this was broken out into a new method?

Reply:
i come from embedded system were we break everything in functions. This is the reason...